Controller/ConnectController.php

```diff
@@ -359,4 +348,60 @@ protected function getTemplatingEngine()
     {
         return $this->container->getParameter('hwi_oauth.templating.engine');
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Symfony <2.6 BC. To be removed.
+     */
+    protected function isGranted($attributes, $object = null)
```

Comment:
This method is here just to keep BC with old Symfony version without adding a if forest on controller code.

This will be removed later and code behavior would not be changed at all.